applications/AdjointFluidApplication/custom_schemes/adjoint_bossak_scheme.h

```diff
@@ -189,41 +161,24 @@ class AdjointBossakScheme : public Scheme<TSparseSpace, TDenseSpace>
             KRATOS_THROW_ERROR(
                 std::runtime_error, "DOMAIN_SIZE != WorkingSpaceDimension", "")
 
-        if (rModelPart.HasSubModelPart(mBoundaryModelPartName) == false)
-        {
-            KRATOS_THROW_ERROR(
-                std::runtime_error,
-                "invalid parameters \"boundary_model_part_name\": ",
-                mBoundaryModelPartName)
-        }
-
         // initialize the variables to zero.
         for (auto it = rModelPart.NodesBegin(); it != rModelPart.NodesEnd(); ++it)
         {
-            it->Set(BOUNDARY, false);
-            it->FastGetSolutionStepValue(SHAPE_SENSITIVITY) = SHAPE_SENSITIVITY.Zero();
             it->FastGetSolutionStepValue(ADJOINT_VELOCITY) = ADJOINT_VELOCITY.Zero();
```

Comment:
also as a comment it would be good if you do some minimal checking, something like

if(  ! rModelPart.NodesBegin()->HasSolutionStepVariable(ADJOINT_VELOCITY) )
       KRATOS_ERROR << "variable ADJOINT_VELOCITY not allocated ...." 

it saves a lot of debugging time...